java/src/main/java/org/micromanager/lightsheet/StackResampler.java

```diff
@@ -53,6 +52,22 @@ public class StackResampler {
    private final boolean maxProjection_;
 
 
+   /**
+    * StackResampler constructor.  Sets up matrices used for the transform, and
+    * pre-calculates the re-mapped coordinate space.
+    *
+    * @param mode YX Projection (0), Orthogonal views (1), or full volume (2)
+    * @param maxProjection Do a maximum intensity projection if true, otherwise returns
+    *                      mean projections.
+    * @param theta Angle with optical axis in radians.
+    * @param cameraPixelSizeXyUm Size of one (square) camera pixel in the object plane
+    *                            in microns.
+    * @param zStepUm Distance (in microns) between two slices of the input stack in the object
```

Comment:
I think this is related to my confusion below. It seems weird to me that we have camera XY and then this stage Z, and the stage Z is not perpendicular to the camera XY axes. Unless it is a convention already to report z step in this way, I would think it would be clearer to make z step perpendicular to sensor and then do appropriate conversions within the class

Reply:
> I think this is related to my confusion below. It seems weird to me that we have camera XY and then this stage Z, and the stage Z is not perpendicular to the camera XY axes. 

Well, that is the whole trick of OPM (Oblique plane Microscopy).  For the experimentalist, I believe that it is most natural to think of the angle theta as the angle with the optical axis, and as "zStep", the distance (in the XY plane) between each consecutive image in the stack (even though these are not orthogonal planes).  You could define both theta and zStep differently, but will need to explain carefully.  It may be wise to rename from "zStep" to "frameDistance", or something like that, i.e. remove "z" from the vocabulary.

Reply:
> It may be wise to rename from "zStep" to "frameDistance", or something like that, i.e. remove "z" from the vocabulary.

Makes sense. You know more about this from user perspective, so feel free to rename as you see fit

Reply:
Done, but made it "sliceDistance";)